src/nupic/encoders/random_distributed_scalar.py

```diff
@@ -217,6 +217,8 @@ def mapBucketIndexToNonZeroBits(self, index):
     Given a bucket index, return the list of non-zero bits. If the bucket
     index does not exist, it is created. If the index falls outside our range
     we clip it.
+
+    @param returns numpy array of indices of non-zero bits for specified index
```

Comment:
Don't you mean `@return numpy array of indices of non-zero bits for specified index`? It would also be useful to have `@param index ...`